src/main/java/io/github/perrymant/moneymaker/FileIO.java

```diff
@@ -0,0 +1,31 @@
+package io.github.perrymant.moneymaker;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+class FileIO {
+
+    static String read(String fileName) throws IOException {
+        BufferedReader text = new BufferedReader(new FileReader(fileName));
+        List<String> data = new ArrayList<>();
+        String s;
+        while ((s = text.readLine()) != null) {
```

Comment:
There is [a lot easier way](https://stackoverflow.com/a/326440/2422278) of reading files than this. E.g.

```java
return new String(Files.readAllBytes(Paths.get(fileName)), Charset.defaultCharset());
```